src/TitleDetails/TitleDetails.js

```diff
@@ -346,6 +361,22 @@ const TitleDetails = ({
     ],
   );
 
+  const hasUnreceivable = Boolean(piecesExistance?.[PIECE_STATUS.unreceivable]);
+  const renderUnreceivablePiecesActions = useCallback((renderColumnsMenu) => (
```

Comment:
Why do you need `useCallback` here?

Reply:
Agree, in this case, it is superfluous. Updated